### PR TITLE
Add devsecops

### PR DESCRIPTION
**Pull request by Charley-Mann**

### CODE REVIEW

File: content/radars/2021-09-devsecops.yml [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    linkedin: peteanusergiu
+    title: Head of DevOps at Allianz Direct
+  - name: Keith Nielsen
+    photo: https://www.cncf.io/wp-content/uploads/2021/09/Keith-Nielsen.jpeg
+    title: Director Cloud Architecture at Discover Financial Services
+points:
+  - name: Artifactory 
+  - homepage: https://jfrog.com/artifactory/
+    level: adopt
+    votes: 16
```
Comment (kgamanji) [inline content/radars/2021-09-devsecops.yml:48]:
```suggestion
     votes: 16
```
The is an extra space that should be in front of "votes" line. In short the indentation is not matching with the lines above

**Comment by Charley-Mann** [comment]
@kgamanji -- still can't seem to get the preview to deploy. Any ideas?

File: content/radars/2021-09-devsecops.yml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    photo: https://www.cncf.io/wp-content/uploads/2021/09/Sergio-Patean.jpeg
+    twitter: PeteanuSergiu
+    linkedin: peteanusergiu
+    title: Head of DevOps at Allianz Direct
+  - name: Keith Nielsen
+    photo: https://www.cncf.io/wp-content/uploads/2021/09/Keith-Nielsen.jpeg
+    title: Director Cloud Architecture at Discover Financial Services
+points:
+  - name: Artifactory 
+  - homepage: https://jfrog.com/artifactory/
```
Comment (kgamanji) [inline content/radars/2021-09-devsecops.yml:46]:
```suggestion
    homepage: https://jfrog.com/artifactory/
```

File: content/radars/2021-09-devsecops.yml (hunk 1 above)
Comment (kgamanji) [reply content/radars/2021-09-devsecops.yml:48]:
```suggestion
    votes: 
```

"votes" is a top-level parameter that includes the number of adopt, trial and asses votes. It should not include the total votes

File: content/radars/2021-09-devsecops.yml [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    photo: https://www.cncf.io/wp-content/uploads/2021/09/Keith-Nielsen.jpeg
+    title: Director Cloud Architecture at Discover Financial Services
+points:
+  - name: Artifactory 
+    homepage: https://jfrog.com/artifactory/
+    level: adopt
+    votes: 16
+      adopt: 14
+      trial: 1
+      hold: 1
```
Comment (kgamanji) [inline content/radars/2021-09-devsecops.yml:51]: We do not show the number of `hold` votes. 
```suggestion
      asses: ??
```

Tho looking at other radars, we seem to have the `hold` votes for some tools. Maybe we could add them for each tool in GitHub, as they will not be showcased on the radar page.

**Comment by kgamanji** [comment]
@Charley-Mann "votes" is a top-level parameter that includes the number of adopt, trial and asses votes. It should not include the total votes. I have left a comment on how to update it. You will need to do it for all other tools

File: content/radars/2021-09-devsecops.yml [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+      trial: 4
+      assess: 3
+    - name: Sonatype Nexus
+    - homepage: https://www.sonatype.com/products/repository-pro
+      level: assess
+    votes: 10
+      adopt: 6
+      trial: 1
+      assess: 0
+companies:
```
Comment (kgamanji) [inline content/radars/2021-09-devsecops.yml:157]: The list only includes 23 organizations. We had 24 end users orgs that submitted their votes. Seems like one is missing from this list

File: content/radars/2021-09-devsecops.yml [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+    homepage: https://jfrog.com/artifactory/
+    level: adopt
+    votes: 16
+      adopt: 14
+      trial: 1
+      hold: 1
+  - name: Sonarqube
+  - homepage: https://www.sonarqube.org/
+    level: adopt
+    votes: 15
```
Comment (kgamanji) [inline content/radars/2021-09-devsecops.yml:55]:
```suggestion
    votes:
```
Editing this one too, as I am not sure the previous comment will be visible.

File: content/radars/2021-09-devsecops.yml (hunk 4 above)
Comment (Charley-Mann) [reply content/radars/2021-09-devsecops.yml:146]: resolved on slack

File: content/radars/2021-09-devsecops.yml (hunk 5 above)
Comment (Charley-Mann) [reply content/radars/2021-09-devsecops.yml]: resolved on slack

File: content/radars/2021-09-devsecops.yml (hunk 3 above)
Comment (Charley-Mann) [reply content/radars/2021-09-devsecops.yml]: resolved on slack

**Comment by Charley-Mann** [comment]
The [build](https://app.netlify.com/sites/cncf-radar/deploys/614ad7402f2cab000708ebbd) has broken when I removed the `draft` line with [the last commit](https://github.com/Charley-Mann/radar/compare/0c07ba49c9385fd1487ea54460c632724860d241...5d89229) but the error message (L75) seems unrelated:
```
10:12:28 AM: $ yarn prepare-radars && next build && next export && yarn post-process
10:12:29 AM: $ babel-node scripts/prepareRadars.js
10:12:34 AM: TypeError: Cannot read property 'numEmployeesMin' of undefined
10:12:34 AM:     at _callee4$ (/opt/build/repo/scripts/prepareRadars.js:69:42)
```

Any ideas @kgamanji @jordinl83 ?

**Pull request merged**